src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -108,7 +108,7 @@
   <Target Name="PreCloudBuild">
     <!-- copy runner scripts so they can be uploaded as supplemental payload -->
     <ItemGroup>
-      <RunnerScripts Include="$(ToolsDir)RunnerScripts/**/*.py" />
+      <RunnerScripts Include="$(ToolsDir)RunnerScripts/**/*.*" />
```

Comment:
Don't just include everything. There will be random pre-compiled things in this folder, and all sorts of temp files.  Even the *.py is pretty dangerous, but anything other than that should be included specifically.